### DISCUSSION TITLE
expandParent prop behavior

### DISCUSSION THREAD

**Discussion by odeyal0** [xyflow/xyflow#2164] [General]
Why expandParent prop only expand parent component, but not narrow down?
Before moving child node:
![image](https://user-images.githubusercontent.com/46008279/169783195-8ffccad6-299e-4ae9-be84-807c4290b08d.png)
After moving child node:
![image](https://user-images.githubusercontent.com/46008279/169783304-aaf92803-1674-43de-a56e-24fcccab829f.png)
![image](https://user-images.githubusercontent.com/46008279/169783443-1d83e28e-9bb6-4df6-8f4b-2efa3efaddde.png)

code:
{ id: 'placeholderNode', position: { x: 0, y: -150 }, type: 'group', isParent: true },

  { id: '0', data: { vertical: true, color: 'red' }, position: { x: 0, y: 0 }, type: 'disconnector', parentNode: 'placeholderNode', expandParent: true },
  { id: '1', data: { vertical: true, color: 'red' }, position: { x: 100, y: 100 }, type: 'disconnector', parentNode: 'placeholderNode', expandParent: true },

**Comment by tth-ops** [comment]
the same question，mark
